scripts/_helpers.py

```diff
@@ -407,6 +407,9 @@ def make_accessable(*ios):
         job.rule.name,
         None,
     )
+
+    snakemake.benchmark = job.benchmark
```

Comment:
Cool. What does it do?

Reply:
It adds the benchmark file to the mock; otherwise the benchmark path is not stored and I cannot collect statistics from the benchmark dynamically with the rule